pytensor/compile/builders.py

```diff
@@ -433,22 +377,25 @@ def __init__(
         self.grad_overrides = grad_overrides
         self.rop_overrides = rop_overrides
 
-        if lop_overrides != "default":
-            if grad_overrides != "default":
+        self._lop_op_constructor = lop_overrides
```

Comment:
Pet peeve: Why the naming inconsistency between the user-facing API and the internals ("override" vs "constructor").

Reply:
Because it's a better name imo, but I don't want to change the user facing API

Reply:
I'll change to use the outside name

Reply:
Done